Services/JobRunnerWebJob/appsettings.Development.json

```diff
@@ -7,12 +7,20 @@
     "ActiveQueueName": "",
     "ServiceUrl": "",
     "ServiceKeyVaultName": "",
-    "AppInsightsIntrumentation": "",
+    "AppInsightsIntrumentationKey": "",
     "MicrosoftAuthority": "",
     "DataHubIdentifier": "",
```

Comment:
we no longer use DataHub in our code base.

Reply:
Updated.